src/utils/airtable.js

```diff
@@ -7,10 +7,8 @@ function getLogo(Logo, LinkedinProfile) {
   return logoSharp?.resize || logoSharp?.fuild || logoSharp?.fixed
 }
 
-function getDescription(description) {
-  const offset = 30
-  const index = description?.slice(offset).search(/\./) ?? -1
-  return index === -1 ? description : description.substr(0, index + offset + 1)
+function getAbout(About, LinkedinProfile) {
+  return LinkedinProfile?.[0]?.data.About || About || ""
```

Comment:
"About" actually pulls from LinkedIn Profiles already using a formula on the Airtable site.

Reply:
Nice, i guess i can simplify this :)